### DISCUSSION TITLE
脚注被导航栏覆盖住有什么好的解决办法吗？

### DISCUSSION THREAD

**Discussion by AmenLi** [HEIGE-PCloud/DoIt#536] [Q&A] [answered]
https://hugodoit.pages.dev/zh-cn/basic-markdown-syntax/#%E5%9B%BE%E7%89%87
比如demo网站里
从文章跳到文末脚注没有问题，如果从文末脚注往文章上跳转，就会被导航盖住。
不知道大家有没有什么好的办法可以解决

**Comment by HEIGE-PCloud** [comment, marked as answer]
修了 #538

**Reply by AmenLi** [reply]
好快，牛逼
